Dockerfile

```diff
@@ -1,25 +1,29 @@
-FROM jenkins:2.0
+FROM jenkins/jenkins:2.239
```

Comment:
As far as I remember there was a breaking change in one of the early Jenkins 2.x releases which has the potential to break a lot of builds.
So if this would be deployed to builds.wetransform.to I would expect that someone needs to spend at least 1-2 days for the update (also because of potential issues with Jenkins plugins)

Reply:
Okay, I wouldn't have expected breaking changes considering the versioning. I'll see if I can downgrade again and still make the chrome installation work 🙄